### PR TITLE
CHANGELOG 5.5.1

### PR DESCRIPTION
**Pull request by directionless**
Changelog for 5.5!

And some bonus commits that got missed in the 5.3.0 changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -89,6 +152,9 @@ This release represents commits from 15 contributors! Thank you all.
 ### Deprecation Notices
 
 - Deprecate unmaintainable legacy table, `smart_drive_info` ([#7464](https://github.com/osquery/osquery/issues/7464), [#7542](https://github.com/osquery/osquery/pull/7542))
+- Drop shortcut_files table ([#7547](https://github.com/osquery/osquery/pull/7547))
+- Drop the ssdeep library and remove its hash support  ([#7525](https://github.com/osquery/osquery/pull/7525))
+- Remove libelfin and elf parsing tables ([#7524](https://github.com/osquery/osquery/pull/7524))
```
Comment (Smjert) [inline CHANGELOG.md:157]: These are already included in 5.2.3; even if 5.2.3 was in a separate branch, 5.3.0 should be still considered as a descendant, so it includes those.

Comment (directionless) [reply CHANGELOG.md:157]: You're right! My grep missed those. Will remove!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,67 @@
 # osquery Changelog
+<a name="5.5.0"></a>
+## [5.5.0](https://github.com/osquery/osquery/releases/tag/5.5.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.4.0...5.5.0)
+
+Osquery 5.5.0 has some really exciting table updates! There is much
```
Comment (mike-myers-tob) [inline CHANGELOG.md:7]: there is _the_ much anticipated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,67 @@
...
+anticipated `unified_log` for macOS, this table is the replacement for
+`asl`, and uses the current Apple APIs. Additionally, several tables
+have improved their cross platform support.
```
Comment (mike-myers-tob) [inline CHANGELOG.md:10]: nit: cross-platform

**Comment by mike-myers-tob** [comment]
Looks good! Feel free to merge it right away but I offered a couple of nits to fix if they can make it in

File: CHANGELOG.md (hunk 2 above)
Comment (directionless) [inline CHANGELOG.md:7]:
```suggestion
Osquery 5.5.0 has some really exciting table updates! There is a much
```

Comment (directionless) [reply CHANGELOG.md:7]: you think "the" not "a" ?

File: CHANGELOG.md (hunk 3 above)
Comment (directionless) [reply CHANGELOG.md:10]:
```suggestion
have improved their cross-platform support.
```

File: CHANGELOG.md (hunk 2 above)
Comment (mike-myers-tob) [reply CHANGELOG.md:7]: either way! But I would write it's "a much anticipated"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,67 @@
 # osquery Changelog
+<a name="5.5.0"></a>
+## [5.5.0](https://github.com/osquery/osquery/releases/tag/5.5.0)
```
Comment (mike-myers-tob) [inline CHANGELOG.md:3]: I don't know how you handled changelog for point-releases in the past but for 5.5.1:

`libs: Update sqlite to version 3.39.2 (https://github.com/osquery/osquery/pull/7736)`

Comment (mike-myers-tob) [reply CHANGELOG.md:3]: And

- `Add cgroup_path column to processes table on Linux (#7728)`
- `Update platform_info schema to handle platform specific columns`
- `Move process_open_sockets.state to normal schema`

Comment (directionless) [reply CHANGELOG.md]: Updated this for 5.5.1. 

Since 5.5.0 never went stable, I'm going to pretend like it never existed

**Comment by mike-myers-tob** [comment]
Closed/re-opened to re-run the failing test fixed in https://github.com/osquery/osquery/pull/7742

**Comment by directionless** [comment]
huh, I thought I closed and re-opened. But I don't see it in the history. Thanks!

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,74 @@
 # osquery Changelog
 
+<a name="5.5.1"></a>
+## [5.5.1](https://github.com/osquery/osquery/releases/tag/5.5.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.4.0...5.5.1)
+
+Osquery 5.5.0 has some really exciting table updates! There is a much
```
Comment (Smjert) [inline CHANGELOG.md:8]: nit: osquery 5.5.1?

**Review by Smjert — APPROVED** [review]
Minor nit, your choice.

File: CHANGELOG.md (hunk 5 above)
Comment (directionless) [reply CHANGELOG.md:8]: Not a nit! Missed one
```suggestion
Osquery 5.5.1 has some really exciting table updates! There is a much
```

**Review by Smjert — APPROVED** [review]

**Pull request merged**